packages/transition-frontend/src/services/routing/RoutingUtils.ts

```diff
@@ -86,3 +86,43 @@ export const calculateAccessibilityMap = async (
         );
     });
 };
+
+export const calculateMapIntersection = async (
```

Comment:
Is this really belong to a RoutingUtil object ? 
See my previous comment, since this might not be the best flow for this.